connect-examples/v2/python_payment/sqpaymentform.js

```diff
@@ -1,9 +1,3 @@
-// Set the application ID
-var applicationId = "REPLACE_ME";
-
-// Set the location ID
-var locationId = "REPLACE_ME";
-
 /*
```

Comment:
just curious is this file `sqpaymentform.js` the same as `php_payment/sqpaymentform.js`?